### ISSUE TITLE
Interface "out-of-sync"

### ISSUE THREAD

**Issue by adelorenzo-portainer** [lllyasviel/Fooocus#703]
I am experiencing a strange issue with the interface. After 4 or 5 image generations the interface freezes and the backend continues. From then onwards Fooocus only displays the previous images that I generated and not the one that it is generating. Killing it and bringing it back up sometimes doesn't fix the problem.

**Comment by MoonRide303** [comment]
@adelorenzo-portainer Did you observe any errors in the log, or used any specific settings / functions before it happened?

**Comment by adelorenzo-portainer** [comment]
nope, none but it happens whet I load Fooocus using tmux and run it remotely.

**Comment by adelorenzo-portainer** [comment]
it happened again but this time I am not running on a tmux terminal. I strarted Fooocus from the terminal on my machine an ran it remotely. no error message.

**Comment by 666spawn999** [comment]
this could be the same as the Queue getting out of sync. 
--> https://github.com/lllyasviel/Fooocus/issues/713

When the websocket loses connection, or you are working with multiple windows the queue gets jumbled up

**Comment by prefetchgif** [comment]
fwiw, i'm seeing this as well.  exact same issue.
if you run in only a single window, it happens less often, but it definitely still happens.
no error appears in the console, but the browser GUI just hangs.  upon a refresh, you lose all your settings for that session and the images displayed become out of sync.

happens on chrome and firefox on osx.  fooocus server is running on a windows machine.

**Comment by lllyasviel** [comment]
hi this is fixed one week ago. feel free to reopen if there is still any problems
